### ISSUE TITLE
WESM.gpkg coordinate ordering changed!

### ISSUE THREAD

**Issue by scottyhq** [uw-cryo/coincident#29]
Very strange, but sometime recently, the coordinate ordering in the USGS WESM.gpkg changed, causing our search and tests to fail 🤦 : 

```
ogrinfo -so -al /vsicurl/https://prd-tnm.s3.amazonaws.com/StagedProducts/Elevation/metadata/WESM.gpkg | grep Extent
#Extent: (13.232000, -179.250067) - (71.507000, 179.854651)
ogrinfo -so -al WESM.gpkg | grep Extent            
#Extent: (-179.250067, 13.232000) - (179.854651, 71.507000)
```

(maybe last update on 2024-11-24T06:33:34.000Z according to https://prd-tnm.s3.amazonaws.com/index.html?prefix=StagedProducts/Elevation/metadata/)

I'll setup a workflow to run tests nightly to keep track better track of these sorts of changes... Also sent an email to USGS.

**Comment by dshean** [comment]
Good catch! Did the crs and axis order definition change? Are we loading that directly, or assuming 4326?

**Comment by scottyhq** [comment]
Quick note that for now I'm just doing this to reverse the coordinates after loading: 
```python
gf_lidar['geometry'] = gf_lidar['geometry'].map(lambda polygon: shapely.ops.transform(lambda x, y: (y, x), polygon))
```

**Comment by scottyhq** [comment]
Seems fixed as of file update at 2024-12-04T20:03:29.000Z
